src/index.ts

```diff
@@ -39,6 +43,10 @@ const removeBookmarkAndHighlight = safe(unsafe_removeBookmarkAndHighlight)
 const isMatchingBookmark = safe(unsafe_isMatchingBookmark)
 const isBookmarkStart = safe(unsafe_isBookmarkStart)
 const isBookmarkEnd = safe(unsafe_isBookmarkEnd)
+const isToggleStart = safe(unsafe_isToggleStart)
```

Comment:
the only point of this safe wrapper is to catch documentEditor errors. If your functions don't use the FE documentEditor instance, then you most likely don't need to do this.

Reply:
updated @AdamTombleson